validator/main.go

```diff
@@ -162,6 +165,7 @@ contract in order to activate the validator client`,
 			formatter := new(prefixed.TextFormatter)
 			formatter.TimestampFormat = "2006-01-02 15:04:05"
 			formatter.FullTimestamp = true
+			formatter.DisableColors = ctx.GlobalString(cmd.LogFileFormat.Name) != ""
```

Comment:
Same thing here, add a comment to explain why we need to disable colors when logging to file.

Reply:
Done